test/test-typescript-diff/expected/output

```diff
@@ -1,5 +1,5 @@
 Stack: hello-diff
 Resources
- + NULL_RESOURCE        test                null_resource.hellodiff_test_85E428D7
+ + NULL_RESOURCE        hellodiff_test                null_resource.hellodiff_test_85E428D7
```

Comment:
I intentionally left this test without the new flags, so ideally this wouldn't change, but it would take some refactoring to get enough information to ResourceName in order to fix this.

Reply:
@skorfmann Any thoughts on this? Basically the cli isn't able to handle pulling the stack name off when not using the new behaviors in the case that there is and underscore or dash in the stack name. We'd need to propagate the feature flag status (or the entire config) all the way to `ResourceName` in order to fix.

Reply:
That's a pure cosmetic UI thing, right?

Reply:
Yeah. Technically someone could be parsing the output, but I wouldn't expect that.

Reply:
Yeah, I think that's not an issue then.